NullPointerException in module uses statement constructor

`new com.github.javaparser.ast.modules.ModuleUsesStmt()`

produces

```java
java.lang.AssertionError: A reference was unexpectedly null.
	at com.github.javaparser.utils.Utils.assertNotNull(Utils.java:55)
	at com.github.javaparser.ast.modules.ModuleUsesStmt.setType(ModuleUsesStmt.java:69)
	at com.github.javaparser.ast.modules.ModuleUsesStmt.<init>(ModuleUsesStmt.java:38)
	at com.github.javaparser.ast.modules.ModuleUsesStmt.<init>(ModuleUsesStmt.java:29)
	at com.github.javaparser.ast.modules.ModuleUsesStmt.<init>(ModuleUsesStmt.java:24)
	...
```

--- a/javaparser-core/src/main/java/com/github/javaparser/ast/modules/ModuleUsesStmt.java
+++ b/javaparser-core/src/main/java/com/github/javaparser/ast/modules/ModuleUsesStmt.java
@@ -4,6 +4,7 @@ import com.github.javaparser.ast.AllFieldsConstructor;
 import com.github.javaparser.ast.Node;
 import com.github.javaparser.ast.nodeTypes.NodeWithType;
 import com.github.javaparser.ast.observer.ObservableProperty;
+import com.github.javaparser.ast.type.ClassOrInterfaceType;
 import com.github.javaparser.ast.type.Type;
 import com.github.javaparser.ast.visitor.CloneVisitor;
 import com.github.javaparser.ast.visitor.GenericVisitor;
@@ -21,7 +22,7 @@ public final class ModuleUsesStmt extends ModuleStmt implements NodeWithType<Mod
     private Type type;
 
     public ModuleUsesStmt() {
-        this(null);
+        this(null, new ClassOrInterfaceType());
     }
 
     @AllFieldsConstructor

